src/routers/bookmark_router.py

```diff
@@ -0,0 +1,140 @@
+from fastapi import APIRouter, Depends, HTTPException
+from typing import List
+from pydantic import BaseModel
+from sqlmodel import Session, select
+
+from authentication import KeycloakUser, get_user_or_raise
+from database.session import get_session
+from database.model.bookmark.bookmark import Bookmark
+from http import HTTPStatus
+from database.model.concept.concept import AIoDConcept
+from database.model.helper_functions import non_abstract_subclasses
+
+
+class BookmarkCreate(BaseModel):
+    resource_identifier: str
+
+
+class BookmarkRead(BaseModel):
+    resource_identifier: str
+    created_at: str
```

Comment:
```suggestion
class BookmarkCreate(BaseModel):
    resource_identifier: str


class BookmarkRead(BookmarkCreate):
    created_at: str
```
Also provide descriptions through a Pydantic `Field` (i.e. `= Field(description=...)`). They will show up in the Swagger UI.